app.py

```diff
@@ -1,8 +1,14 @@
+import constants
+import window as w
```

Comment:
Notice here that I'm importing "window.py", but since there's already a window variable in this file, I aliased the import to "w", so we can do `w.whatever`